tests/page-objects/display/languages.wdio.page.js

```diff
@@ -0,0 +1,62 @@
+const utils = require('../../utils');
+
+const submitButton = () => $('div.form-actions button.btn.btn-primary.ng-scope');
```

Comment:
I think almost all elements on the page have the `ng-scope` class. I think we can safely remove it from selectors, because it just clutters.